binding.gyp

```diff
@@ -119,9 +119,9 @@
         ["OS=='win'", {
           "conditions": [
             ["target_arch=='ia32'", {
-              "openssl_root%": "C:/OpenSSL-Win32"
+              "openssl_root%": "<(module_root_dir)/nuget/openssl-vc141-static-x86_64.1.0.2/build/native/lib/Win32"
```

Comment:
Does this mean windows users will have to install openssl with nuget?

Reply:
Hmm we could make `nuget` install the package to the same location as `OpenSSL` so we don't need to change this.

I will investigate if that's possible.

Reply:
That would be preferable. I'd like to avoid extra work for windows users.